Userland/Libraries/LibC/wchar.cpp

```diff
@@ -61,6 +61,19 @@ wchar_t* wcscpy(wchar_t* dest, const wchar_t* src)
     return original_dest;
 }
 
+wchar_t* wcsdup(const wchar_t* str)
```

Comment:
What do you think about starting to add spec links to LibC, like we do in `LibJS`, `LibWeb`? 
@IdanHo started doing it in his change where he added `sigtimedwait(..)`, maybe we should continue? 

```cpp
// https://pubs.opengroup.org/onlinepubs/9699919799/functions/wcsdup.html
wchar_t* wcsdup(const wchar_t* str)
{
...
}

Reply:
:thinking: I like it
I'm working on implementing a bunch of these, I'll throw a "add spec links" commit in

Reply:
I haven't considered it yet, as the "spec" usually isn't too helpful for structuring the implementation (at least in comparison to what I have seen of `LibJS` and `LibWeb`). I also usually just go by what `man 3p` says instead of looking up the specification online :^)